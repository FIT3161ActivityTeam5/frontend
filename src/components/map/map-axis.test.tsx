import React from 'react';
import renderer from 'react-test-renderer';

import MapAxis from './map-axis';

describe('<MapAxis />', () => {
    it('renders correctly', () => {
      const tree = renderer.create(
          <MapAxis
              canvasSize={8.0}
          />
      ).toJSON();
      expect(tree).toMatchSnapshot();
    });
    it('applies stroke width correctly', () => {
      const tree = renderer.create(
          <MapAxis
            canvasSize={8.0}
            thickness={4.0}
          />
      ).toJSON();
      expect(tree).toMatchSnapshot();
    });
});
