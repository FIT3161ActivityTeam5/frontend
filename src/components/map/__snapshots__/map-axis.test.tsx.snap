// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`<MapAxis /> applies stroke width correctly 1`] = `
Array [
  <RNSVGCircle
    cx={4}
    cy={4}
    propList={
      Array [
        "stroke",
        "strokeWidth",
      ]
    }
    r={128}
    stroke={4291941851}
    strokeWidth={4}
  />,
  <RNSVGCircle
    cx={4}
    cy={4}
    propList={
      Array [
        "stroke",
        "strokeWidth",
      ]
    }
    r={384}
    stroke={4291941851}
    strokeWidth={4}
  />,
  <RNSVGLine
    propList={
      Array [
        "stroke",
        "strokeWidth",
      ]
    }
    stroke={4291941851}
    strokeWidth={4}
    x1={4}
    x2={4}
    y1={0}
    y2={8}
  />,
  <RNSVGLine
    propList={
      Array [
        "stroke",
        "strokeWidth",
      ]
    }
    stroke={4291941851}
    strokeWidth={4}
    x1={0}
    x2={8}
    y1={4}
    y2={4}
  />,
]
`;

exports[`<MapAxis /> renders correctly 1`] = `
Array [
  <RNSVGCircle
    cx={4}
    cy={4}
    propList={
      Array [
        "stroke",
        "strokeWidth",
      ]
    }
    r={128}
    stroke={4291941851}
    strokeWidth={2}
  />,
  <RNSVGCircle
    cx={4}
    cy={4}
    propList={
      Array [
        "stroke",
        "strokeWidth",
      ]
    }
    r={384}
    stroke={4291941851}
    strokeWidth={2}
  />,
  <RNSVGLine
    propList={
      Array [
        "stroke",
        "strokeWidth",
      ]
    }
    stroke={4291941851}
    strokeWidth={2}
    x1={4}
    x2={4}
    y1={0}
    y2={8}
  />,
  <RNSVGLine
    propList={
      Array [
        "stroke",
        "strokeWidth",
      ]
    }
    stroke={4291941851}
    strokeWidth={2}
    x1={0}
    x2={8}
    y1={4}
    y2={4}
  />,
]
`;
